e2e/nomostest/nt.go

```diff
@@ -711,6 +713,26 @@ func (nt *NT) portForwardPrometheus() {
 		prometheusServerDeploymentName,
 		nt.prometheusPortForwarder,
 	)
+
+	if err := nt.emptyPrometheusCache(); err != nil {
+		nt.T.Fatalf("failed to empty prometheus cache: %v", err)
+	}
+}
+
+// emptyPrometheusCache empty the cache in Prometheus.
+func (nt *NT) emptyPrometheusCache() error {
+	nt.T.Helper()
+	ctx, cancel := context.WithCancel(nt.Context)
+	defer cancel()
+
+	c, err := prometheusapi.NewClient(prometheusapi.Config{
+		Address: fmt.Sprintf("http://localhost:%d", prometheusServerPort),
```

Comment:
This is using the port forward on localhost, so you'll need to get the port from `nt.prometheusPortForwarder.LocalPort()` (and you'll probably want to wrap this in a retry loop)

Reply:
Good catch. Done.